packages/db/src/migration-scripts/migrate-to-branching.ts

```
@@ -0,0 +1,322 @@
+                            .where(inArray(frames.id, chunk));
+                    });
+                }
+            }
+        }
+
+        // Step 6: Verification
+        console.log('✅ Verifying migration completeness...');
+
+        const framesWithoutBranches = await db
+            .select({ count: frames.id })
+            .from(frames)
```

Biome (2.1.2)

[error] Shouldn't redeclare 'framesWithoutBranches'. Consider to delete it or rename it.

'framesWithoutBranches' is defined here:


(lint/suspicious/noRedeclare)